docs/guide/topics-cache.md

```diff
@@ -16,7 +16,7 @@ return [
 ];
 ```
 
-If you only use the redis cache, you can also configure the parameters of the connection within the
+If you use redis only as a cache, you can also configure the parameters of the connection within the
```

Comment:
Well, actually I was trying to clarify the same thing.
I thought that "If you only use the redis cache" could be misunderstood as "only the redis cache, i.e. without using  APC, memcache, file cache, etc.".

But, OK, I will try to rewrite it.